.travis.yml

```diff
@@ -9,7 +9,7 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
-install: yarn install
+install: yarn install && yarn run build
```

Comment:
Travis needs to build during the publish process, otherwise the lib/dist folders don't get created